packages/browser/src/integrations/browsersession.ts

```diff
@@ -0,0 +1,39 @@
+import { addHistoryInstrumentationHandler } from '@sentry-internal/browser-utils';
+import { captureSession, defineIntegration, logger, startSession } from '@sentry/core';
+import { DEBUG_BUILD } from '../debug-build';
+import { WINDOW } from '../helpers';
+
+/**
+ * When added, automatically creates sessions which allow you to track adoption and crashes (crash free rate) in your Releases in Sentry.
+ * More information: https://docs.sentry.io/product/releases/health/
+ *
+ * Note: In order for session tracking to work, you need to set up Releases: https://docs.sentry.io/product/releases/
+ */
+export const browserSessionIntegration = defineIntegration(() => {
+  return {
+    name: 'BrowserSession',
+    setupOnce() {
+      if (typeof WINDOW.document === 'undefined') {
+        DEBUG_BUILD &&
+          logger.warn('Using the `browserSessionIntegration` in non-browser environments is not supported.');
+        return;
+      }
+
+      // The session duration for browser sessions does not track a meaningful
+      // concept that can be used as a metric.
+      // Automatically captured sessions are akin to page views, and thus we
+      // discard their duration.
+      startSession({ ignoreDuration: true });
```

Comment:
I had the same thoughts. I am actually extremely unsure on how to proceed. It all depends on how we would want sessions to work. If we want there to be multiple at once, it would make sense, however I would err towards the exact opposite.

I think always only having one session active per page makes the most sense. Even in MFEs. I would even go so far to lock down the API down even more in v9, simply because it makes wrapping your head around the concept easier.

All of this is just thinking out loud though. In general I didn't want this PR to be behaviorally significant. That's why I chose this route.

Reply:
this makes sense to me!

Reply:
I agree, let's leave it at one "global" session per page.